ArmPkg/Library/ArmArchTimerLibEx/ArmArchTimerLibEx.inf

```diff
@@ -0,0 +1,32 @@
+#/** @file
+#
+#  Copyright (c) 2011 - 2014, ARM Limited. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#**/
+

```

Comment:
Probably want to add an override tag for this:https://github.com/microsoft/mu_basecore/blob/dev/202405/BaseTools/Plugin/OverrideValidation/ReadMe.md